LoopMusic/Source/ViewControllers/Settings/SettingViews/DoubleTextFieldOptionalSettingView.swift

```diff
@@ -7,7 +7,7 @@ class DoubleTextFieldOptionalSettingView: BaseSettingViewGeneric<Double?, UIText
         if let number = setting.pointee {
             let format = NumberFormatter()
             format.minimumFractionDigits = 0
-            format.maximumFractionDigits = 4
+            format.maximumFractionDigits = 5
```

Comment:
This should probably use `NumberUtils` to avoid changing the number of digits in two places.

Reply:
Resolved in 9a36244